ext/http/http_next.rs

```diff
@@ -245,9 +245,7 @@ pub fn op_http_get_request_header(
 }
 
 #[op]
-pub fn op_http_get_request_headers(
-  slab_id: SlabId,
-) -> Vec<(ByteString, ByteString)> {
+pub fn op_http_get_request_headers(slab_id: SlabId) -> Vec<ByteString> {
   let http = slab_get(slab_id);
   let headers = &http.request_parts().headers;
   let mut vec = Vec::with_capacity(headers.len());
```

Comment:
This should be doubled, or maybe even slightly overallocated to avoid reallocs when there's "cookies" header

Reply:
Updated. No need to preallocate for cookies, as this header is created as a part of headers map iteration, which mean that if there is one, it'll skipped the initial header `push` and will be added later. This mean that there can be max `headers.len() * 2` entries, even with cookies.